src/ipc/ipc-zephyr.c

```diff
@@ -242,6 +244,8 @@ enum task_state ipc_platform_do_cmd(struct ipc *ipc)
 #endif /* CONFIG_PM  */
 	}
 
+	mem_hot_path_stop_watching();
+
```

Comment:
What if we power down on L243 and we have context save enabled in the build? Wouldn't that mess the tracker?

Reply:
@kv2019i not sure why it should confuse it? The tracker is just a couple of simple variables, if the context is saved, then those variables are saved too. But in fact if any of those power on / off functions are cold, then it might trigger a bug. Let me reduce the watched scope